backends/gstreamer/src/media_stream.rs

```diff
@@ -53,18 +54,23 @@ impl GStreamerMediaStream {
             StreamType::Audio => &*RTP_CAPS_OPUS,
             StreamType::Video => &*RTP_CAPS_VP8,
         };
-        println!("atttaching a {:?} stream", self.type_);
         self.elements.last().as_ref().unwrap().link_filtered(webrtcbin, caps).unwrap();
+        pipeline.set_state(gst::State::Playing).into_result().unwrap();
```

Comment:
is this necessary? the program crashes (with the state change unwrap in prepare_for_negotiation) unless i remove this line

Reply:
So when I remove this line it throws up one blank video screen, i.e. it still doesn't work, but it doesn't panic either.

Reply:
Testing the patch against the JS example has it connect and turn on my camera, but the JS side doesn't show anything and the testsink doesn't show up either.